src/Core/Image/Uploader/Exception/UploadedProductImageConstraintException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Image\Uploader\Exception;
```

Comment:
Also missing license header

Reply:
And the `declare(strict_types=1);`

Reply:
> eclare(strict_types=1);

Ok. If this is a mandatory rule, I think we should add this rule to phpstan.

Reply:
It's totally possible to apply rules only on new files.
See here: https://phpstan.org/user-guide/baseline